include/vkd3d_shader.h

```diff
@@ -86,10 +86,12 @@ struct vkd3d_shader_meta
     vkd3d_shader_hash_t hash;
     unsigned int cs_workgroup_size[3]; /* Only contains valid data if uses_subgroup_size is true. */
     unsigned int patch_vertex_count; /* Relevant for HS. May be 0, in which case the patch vertex count is not known. */
-    unsigned int cs_required_wave_size; /* If non-zero, force a specific CS subgroup size. */
+    unsigned int cs_wave_size_min; /* If non-zero, minimum or required subgroup size. */
```

Comment:
Given how this will bloat our caches, we should pack this in uint8_t x 3 (+1 pad) I think. MaxWaveSize fits fine in 8-bit.

Reply:
Fair, I had considered this but then didn't because the rest also uses larger types than necessary.